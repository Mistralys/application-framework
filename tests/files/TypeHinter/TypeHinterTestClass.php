<?php
/*

class TypeHinterTestClass
{
    public function _handleActions()
    {

    }

    public function _handleActions(){}

    public function _handleActions(?array $param=null)
    {

    }

    public function _handleActions() : bool
    {

    }

    public function _handleActions() : bool
    {
        foreach($variable as $var)
        {
             $doSomething = '';
        }
    }

    public function _handleActions()
    {
        foreach($variable as $var)
        {
             $doSomething = '';
        }
    }

    public function getDefaultAction() : string
    {
        return null;
    }
}
*/
