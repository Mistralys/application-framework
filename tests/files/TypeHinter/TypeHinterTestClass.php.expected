<?php

class TypeHinterTestClass
{
    public function _handleActions() : bool
    {

    }

    public function _handleActions() : bool
    {}

    public function _handleActions(?array $param=null) : bool
    {

    }

    public function _handleActions() : bool
    {

    }

    public function _handleActions() : bool
    {
        foreach($variable as $var)
        {
             $doSomething = '';
        }
    }

    public function _handleActions() : bool
    {
        foreach($variable as $var)
        {
             $doSomething = '';
        }
    }

    public function getDefaultAction() : string
    {
        return '';
    }
}
